step/step.go

```diff
@@ -86,16 +93,10 @@ func (r *ReportGenerator) ProcessConfig() (*Config, error) {
 
 func (r *ReportGenerator) InstallDependencies() error {
 	r.logger.Println()
-	r.logger.Infof("Installing XCTestHTMLReport")
```

Comment:
Can you add this info log line back? The info lines act as a section header for all of the following printf logs.